app/controllers/project.py

```diff
@@ -18,7 +19,6 @@
 from flask_jwt_extended import jwt_required, get_jwt_identity, get_jwt_claims
 from app.helpers.db_flavor import get_db_flavour
 from app.schemas.monitoring_metrics import BillingMetricsSchema
-from app.misc.mongo_ops import save_document, ACTION_LOG
```

Comment:
I think since you removed this (importing save_document and ACTION_LOG) it gives an error on line 129.

Reply:
since  save_document is being called in line 129

Reply:
ditto on this @Mubangizi

Reply:
Hey @bashaashma  Thank you for this observation